tests/bearlib/abstractions/LinterTest.py

```diff
@@ -872,12 +872,17 @@ def test_capture_groups_warnings(self):
             @linter('some-executable',
                     use_stdout=True,
                     output_format='regex',
-                    output_regex=r'(\w+)')
+                    output_regex=r'(?P<not_supported_name>)\d+(\w+)')
```

Comment:
I think you've meant `(?P<not_supported_name>\d+), but that doesn't matter for the test :)